geode-core/src/test/java/org/apache/geode/internal/cache/versions/AbstractVersionTagTestBase.java

```diff
@@ -31,6 +64,122 @@ public void setup() {
     this.vt = createVersionTag();
   }
 
+  int getRandomUnusedInt() {
+    int unusedInt;
+    do {
+      unusedInt = random.nextInt(60000);
+    } while (usedInts.contains(unusedInt));
```

Comment:
this method has failed to add the new `unusedInt` value to `usedInts` before returning

Reply:
oooh - I'll fix that.  I hate flaky tests